packages/docs/pages/reference/elements/doubleflat.mdx

```diff
@@ -8,12 +8,19 @@ import {
 
 # `<doubleflat>`
 
-Short description of what the `<doubleflat>` tag does.
+The self-closing `<doubleflat/>` tag displays the doubleflat symbol in music notation. 
 
 ## Syntax
 
 <SyntaxDisplay name="doubleflat" />
 
 ## Examples
 
-    
\ No newline at end of file
+```ptx-example
+<FRAGMENT parents="p">
+<p>
```

Comment:
Since the fragment has a `p` listed in the `parents`, you should not put a `<p>` tag here (lest you create nested `p`'s in the example.